.firebaserc

```diff
@@ -233,4 +233,4 @@
       }
     }
   }
-}
\ No newline at end of file
+}
```

Comment:
@alexey-petrov remove this change to merge the PR. without it, you won't need @sparkletown/engineering-leads approval